main.go

```diff
@@ -91,24 +121,14 @@ func main() {
 		log.Info().Str("file", cli.Configfile).Msgf("Reading %s", types.Green(cli.Configfile))
 		conf := ReadConfigfile(cli.Configfile)
 
-		// Github
-		repos := github.Get(conf)
-		Backup(repos, conf)
-
-		// Gitea
-		repos = gitea.Get(conf)
-		Backup(repos, conf)
-
-		// Gogs
-		repos = gogs.Get(conf)
-		Backup(repos, conf)
-
-		// Gitlab
-		repos = gitlab.Get(conf)
-		Backup(repos, conf)
-
-		//Bitbucket
-		repos = bitbucket.Get(conf)
-		Backup(repos, conf)
+		if conf.Cron != "" {
```

Comment:
Perhaps this could be a function `HasValidCron`?

```go
func HasValidCron(cronSpec string){
    return cronSpec != ""
}
```

to start. Does the `cron` library have a validation mechanism? Seems so, through [creating a new parser](https://pkg.go.dev/github.com/robfig/cron?utm_source=godoc#NewParser) and checking for an error on [`Parse()`](https://pkg.go.dev/github.com/robfig/cron?utm_source=godoc#Parser.Parse).

Reply:
Could also use that resulting parser to emit a log message confirming the cron spec in some human-readable format...